rn/release-information/release-notes-20-12-update1.adoc

```diff
@@ -0,0 +1,98 @@
+For more information, see <<_defender_name_collisions>>.
+
+// #25872
+* Enables Defender-to-Console network connection configuration (port and proxy) for Defender deployments in VMware Tanzu Application Service (TAS).
+
+// #25814
+* Extends the WAAS proxy to support localhost.
+As part of this change, you can now configure an external port (where WAAS listens) and an internal port (where WAAS forwards requests).
+
+// #25801
+* Fixes a UI issue where charts with insufficient data appeared corrupted.
+
+// #25569
+* Fixes an issue in the UI where the table of credentials couldn't be sorted by credential type.
+
+// #25644
+* Adds support to admission control (Open Policy Agent, OPA) for exec'ing or attaching to a pod.
+To use this new capability, you must redeploy the admission webhook.
+
+// #26087, #25868, #25864, #25669
+* Updates open source packages used in Prisma Cloud Compute components.
+
+
+[#_defender_name_collisions]
+=== Defender name collisions and upgrade considerations
+
+The name assigned to a Defenders has been the name of the underlying host where it runs.
+It's been assumed that hostname is a unique value.
+However, on some cloud platforms, hostname isn't unique.
+Hostnames can be duplicated across environments, causing naming conflicts for deployed Defenders.
+For example, in AWS if you have VPCs with the same network ranges, nodes in EKS clusters can have the same hostname because hostnames are derived from the private ip address of the instance.
+This results in Defenders with duplicate names.
+
+Update 1 fixes the issue by extracting unique context from the cloud metadata, and wrapping hostnames with this context to generate unique hostnames.
+
+
+==== Defender naming scheme
+
+The Defender naming scheme has been updated to guarantee uniqueness.
+
+In AWS, the Defender name is derived as follows: `<hostname>-<instance ID>`.
+For example, `ip-172-31-80-18.ec2.internal-i-07dcaXXXXXXXXXXXX`.
+
+In Azure, the Defender name is dervived as follows: `<hostname>-<resource group name>-<subscription ID>`.
+For example, `aks-agentpool-16600000-vmss000000-MC_alexs-aks_alexs-aks_eastus-ae0XXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX`.
+
+
+==== Enabling the new naming scheme on upgrade
+
+You can configure Prisma Cloud to rename Defenders with unique strings on upgrade.
+Renaming Defenders might break other product configurations, however, so it is disabled by default.
+
+To enable Defender renaming on upgrade, redeploy Defenders in your EKS (AWS) and AKS (Azure) clusters.
+
+. Generate the Defender DaemonSet YAML from the Console UI or with twistcli.
+
+. Open the YAML file, and set the `CLOUD_HOSTNAME_ENABLED` environment variable to `true`.
++
+----
+- name: CLOUD_HOSTNAME_ENABLED
+  value: "true"
+----
+
+. Update the Prisma Cloud objects.
+
+  $ kubectl apply -f twistlock_console.yaml
+
+Upon upgrade, old Defenders appear in Console as disconnected for some time until they're automatically removed by Console.
```

Comment:
Upgraded defenders with a longer unique name appear along with the old disconnected defenders.

Reply:
Thanks for the clarification. This is fixed now.